src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -1312,7 +1312,31 @@ GenTree* Lowering::LowerHWIntrinsic(GenTreeHWIntrinsic* node)
         case NI_AdvSimd_FusedMultiplyAddScalar:
             LowerHWIntrinsicFusedMultiplyAddScalar(node);
             break;
+        case NI_Sve_ConditionalSelect:
+        {
+            LIR::Use use;
+            bool     foundUse = BlockRange().TryGetUse(node, &use);
+
+            JITDUMP("lowering ConditionalSelect HWIntrinisic (before):\n");
+            DISPTREERANGE(BlockRange(), node);
+            JITDUMP("\n");
+
+            GenTree* cndSelNode = LowerHWIntrinsicCndSel(node);
 
+            JITDUMP("lowering ConditionalSelect HWIntrinisic (after):\n");
+            DISPTREERANGE(BlockRange(), cndSelNode);
+            JITDUMP("\n");
+
+            if (cndSelNode != node)
+            {
+                if (foundUse)
+                {
+                    use.ReplaceWith(cndSelNode);
+                }
+                BlockRange().Remove(node);
+            }
```

Comment:
nit: At least on xarch we typically include this as part of the main logic doing the lowering (i.e. `LowerHWIntrinsicCndSel`), as otherwise we're not actually completing the lowering as is, and it can't be called/reused elsewhere if appropriate.

Reply:
fixed.